### PR TITLE
Add Nagios

### PR DESCRIPTION
**Pull request by SwannCroiset**

### CODE REVIEW

File: classes/system/openstack/common/mk20_stacklight_basic.yml [hunk 1]
```diff
@@ -33,11 +33,12 @@ parameters:
           address: 172.16.10.107
           names:
           - mon
+          - nagios
```
Comment (elemoine) [inline classes/system/openstack/common/mk20_stacklight_basic.yml]: Do we really want that?

Comment (SwannCroiset) [inline classes/system/openstack/common/mk20_stacklight_basic.yml]: not really, I find it more elegant :)

Comment (elemoine) [inline classes/system/openstack/common/mk20_stacklight_basic.yml]: We don't do it for elasticsearch and influxdb, so I would not make nagios a special case.

File: classes/system/nagios/server/single.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+applications:
```
Comment (cznewt) [inline classes/system/nagios/server/single.yml]: this file belong to formula/metadata/server/single.yml

File: classes/system/nagios/server/single.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  - nagios
+classes:
```
Comment (cznewt) [inline classes/system/nagios/server/single.yml]: only this classes reference remain

File: classes/system/nagios/server/single.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+- service.nagios.server.single
+parameters:
+  _param: {}
```
Comment (cznewt) [inline classes/system/nagios/server/single.yml]: on this level add param to support heka_alarm collection, look at grafana for example how to set this https://github.com/tcpcloud/salt-formula-grafana/ the remote_data with the engine.

Comment (SwannCroiset) [inline classes/system/nagios/server/single.yml]: I'll do once I've done the corresponding state

**Pull request merged**